graph/src/components/store.rs

```diff
@@ -1780,9 +1780,7 @@ impl AttributeNames {
         match (self, other) {
             (All, All) => {}
             (self_ @ All, other @ Select(_)) => *self_ = other,
-            (Select(_), All) => {
-                unreachable!()
-            }
+            (Select(_), All) => {} // no-op, since we want to keep selected columns
```

Comment:
I thought having the `unreachable` there was better, since we do think this is unreachable, and it's better to have this fail loudly if we ever change the code by mistake so that this becomes reachable.